jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -107,7 +107,7 @@ blockchain_configuration:
   #
   # if not set there is no monetary creation
   # once emptied, there is no more monetary creation
-  total_reward_supply: 100000000000000
+  total_reward_supply: 701917808520000
```

Comment:
I don't understand why you needed to change this value?

Reply:
This is the limit that will be set for the testnet.  It gives a maximum lifetime of 6 months at the agreed return rate

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.

Reply:
See above.